### ISSUE TITLE
IndentationConsistency & IndentationWidth auto-correct combined breaks code

### ISSUE THREAD

**Issue by andreaseger** [rubocop/rubocop#1118]
another bug found in out huge and old codebase.

Before:

``` ruby
module Bar
class Goo
  def something
    first call
      do_other 'things'
      if other > 34
        more_work
      end
  end
end
end
```

After `rubocop --only IndentationConsistency,IndentationWidth --auto-correct indentation_minimal_bug1.rb`

``` ruby
module Bar
  class Goo
    def something
      first call
      do_other 'things'
      if other > 3  4
      more_wor  k
    end
    end
  end
end
```

If I run each of these cops separately one after the other it works as expected and generated the following:

``` ruby
module Bar
  class Goo
    def something
      first call
      do_other 'things'
      if other > 34
        more_work
      end
    end
  end
end
```
